### PR TITLE
Move Client SDK ref guide notebook to md docs

### PR DESCRIPTION
**Pull request by VigdisIS**
Home - Quickstart - Reference guide - Data model

- [x] Add `index.md` with purpose of the SDK
- [x] Add nav for other pages
- [x] Add explanation of entities - describe the process a typical customer has through these
- [x] Getting started: running it locally, azure functions to run functions

Closes [#130](https://github.com/volur-ai/volur-ai-sdk-internal/issues/36)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,7 +1,46 @@
-# Volur AI SDK Documentation
+# Völur AI SDK Overview
 
-This documentation is for the Volur AI SDK. The Volur AI SDK is a Python
-package that provides a simple interface for interacting with the Volur AI API.
+Welcome to the official documentation for the Völur SDK. This Software
+Development Kit (SDK) is designed to provide developers with a robust,
+user-friendly interface for interacting with Völur's API. Our goal is to
+simplify the process of sending data to Völur, enabling you to focus on building
+and optimizing your applications.
```
Comment (nikitabarskov) [inline docs/index.md:7]: Feel free to resolve.

> Our goal is to
simplify the process of sending data to Völur, enabling you to focus on building
and optimizing your applications.

Not sure that what we actually do this :D

File: docs/index.md [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
 
-- [Installation](installation.md),
-- [Getting started](getting-started.md).
```
Comment (nikitabarskov) [inline docs/index.md:7]: I would actually keep the basic links, maybe we can make it even better with https://squidfunk.github.io/mkdocs-material/reference/grids/

Non blocking though.

File: docs/index.md [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+systems, reducing the need for extensive modifications.
+- **Comprehensive Documentation**: Access detailed guides, examples, and
+references to help you get started and troubleshoot any issues.
+
+## Getting Started
+
+Ready to dive in? The SDK documentation is organized into several key sections
+to help you get started:
+
+- **[Quickstart](quickstart.md)**: A concise guide to running an example of the
```
Comment (nikitabarskov) [inline docs/index.md:34]: Could you please merge `quickstart.md` and `getting-started.md`, so we have one source for beginning?

File: docs/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            ),
+        ],
+    )
+
+    # Upload materials information
+    await client.upload_materials_information(source)
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
```
Comment (nikitabarskov) [inline docs/quickstart.md:87]: This is outdated example, please take a look at `examples/main.py`

File: mkdocs.yml [hunk 5]
```diff
@@ -35,11 +35,12 @@ markdown_extensions:
   - pymdownx.tabbed:
       alternate_style: true
 nav:
-  - Getting started: getting-started.md
-  - Installation: installation.md
-  - Authentication: authentication.md
-  - Examples: examples/
-  - API Reference: reference/
```
Comment (nikitabarskov) [inline mkdocs.yml:42]: Authentication and examples are gone :(

**Review by nikitabarskov — COMMENTED** [review]
Looks good!

We should continue to work on structure, and then merge. There are few comments, but some of them are feel free to resolve.

File: docs/deploying.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# Deploying with Azure Functions
```
Comment (rebwibsey) [inline docs/deploying.md:1]: Should we just say that Azure function is one option. There are lots of other options if on for instance are not using Azure. However, it is a good option if one are already using Azure

**Review by rebwibsey — COMMENTED** [review]
Looks good. I would shorten the "Deploying with Azure Functions" part and also noting that this is one of several options for deployment.

**Comment by rebwibsey** [comment]
@VigdisIS this PR has some overlap with [this](https://github.com/volur-ai/python-volur-sdk/pull/71/files). I suggest we add the additional things from that PR into this one and close the other one. Do you agree?

File: docs/deploying.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+### Integrated Security
+Leveraging Microsoft's robust security framework, Azure Functions provides
+built-in features such as authentication and authorization, ensuring that your
+applications are secure from the get-go.
+
+### Event-driven Execution
+Azure Functions supports triggering from a variety of sources, including HTTP
+requests, database operations, and queue messages. This makes it an ideal
+platform for building responsive, event-driven applications using the Volur SDK.
```
Comment (nikitabarskov) [inline docs/deploying.md:36]: Please remove this part.

**Review by nikitabarskov — COMMENTED** [review]
It looks great 👍🏻, just a small comment from my side about Azure Functions.

**Review by rebwibsey — APPROVED** [review]
Looks good. Just note I forgot about my comment and merged [this](https://github.com/volur-ai/python-volur-sdk/pull/71) PR. Feel free to remove things from there if you feel things overlap to much or we can keep both.

**Review by nikitabarskov — APPROVED** [review]

**Pull request merged**